site/blog/cql-starter/cql-quickstart-cql.md

```diff
@@ -325,17 +328,16 @@ select * from starter.cqlstarter;
 
 You should see the single rampup entry along main operation entries in the Cassandra table.
 
-![alt_text](images/image2.png "image_tooltip")
 
-### Customize
+### 3. Customize
 
 Now, let’s customize the cql-starter to make it a bit more your own.
 
-1. Save the .yaml file to your local environment.
-2. Edit the file and uncomment under the default scenario the following entry:
+#### 1. Save the .yaml file to your local environment.
+#### 2. Edit the file and uncomment under the default scenario the following entry:
 
 
-```
+```yaml
    # rampdown: run driver=cql tags==block:rampdown threads==1 cycles==UNDEF
 ```
 
```

Comment:
`target the file it outside` - `it` is redundant here.

Reply:
Good catch!